Lib/fontmake/__main__.py

```diff
@@ -156,6 +156,14 @@ def main(args=None):
         'E.g.: -i "Noto Sans Bold"; or -i ".* UI Condensed". '
         "(for Glyphs or MutatorMath sources only). ",
     )
+    outputGroup.add_argument(
+        "--use-mutatormath",
```

Comment:
maybe add a test case in `tests/test_main.py` that uses this `--use-mutatormath` option and verifies it calls mutatormath